packages/azure-themes/src/azure/styles/Label.styles.ts

```diff
@@ -10,7 +10,7 @@ export const LabelStyles = (props: ILabelStyleProps): Partial<ILabelStyles> => {
   return {
     root: [
       {
-        fontSize: FontSizes.size13,
+        fontSize: FontSizes.size14,
```

Comment:
I'm curious, why did you change labels from 13 to 14? In the design files they're set to 13.

Also, should we the `theme` fontsizes here too?

Reply:
Great catch @wsmd - after speaking with framework design team about font size discrepancy, they concluded that labels on controls are 13px and section headers (such as within forms) are H2, 14px semibold. I will revert our labels to regular theme font size 13 and see if they will be using labels or a different control for form headers!